packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -806,35 +806,24 @@ export class StripeHelper {
    * Returns a boolean indicating success.  It does not throw any exceptions as
    * this operation should not block any functionality.
    *
-   * Currently this only works on customers who use Stripe as their payment
-   * provider.
-   *
    * This will _overwrite_ any existing customer address.
    */
-  async setCustomerLocation(customer: Stripe.Customer): Promise<boolean> {
+  async setCustomerLocation({
+    customerId,
+    postalCode,
+    country,
+  }: {
+    customerId: string;
+    postalCode: string;
+    country: string;
+  }): Promise<boolean> {
```

Comment:
I changed this because a customer does not have a default payment method prior to the first successful subscription.  So the options were either to fetch the customer after the subscription creation and then pass it in here, or use the payment method object that's already available.